backend/game.ts

```
@@ -1,218 +1,239 @@
+            /* --- keep ball outside paddle so it doesn’t “stick” ------- */
+            if (side === 'left') this.ball.x = p.x + p.w + this.ball.r;
+            else this.ball.x = p.x - this.ball.r;
+        });
+
+        /*5) Scoring*/
+        if (this.ball.x + this.ball.r < 0) {
+            // right player scores
+            this.scores.right++;
+            this.spawnTime = Date.now();
+            if (this.scores.right >= WIN_SCORE) return this.end('right');
+            this.reset('right');
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)